sdk/inc/azure/core/az_log.h

```diff
@@ -76,11 +76,20 @@ typedef void (*az_log_message_fn)(az_log_classification classification, az_span
  * @brief Allows the application to specify which #az_log_classification types it is interested in
  * receiving.
  *
- * @details If no classifications are set (`NULL`), the application will receive log messages for
- * all #az_log_classification values.
+ * @param[in] classifications __[nullable]__ An array of #az_log_classification values, terminated
+ * by #AZ_LOG_END_OF_LIST.
  *
- * @param[in] classifications __[nullable]__ An array of az_log_classification values, terminated by
- * #AZ_LOG_END_OF_LIST.
+ * @detail If no classifications are set (\p classifications is `NULL`), the application will
+ * receive log messages for all #az_log_classification values.
+ * @detail If \p classifications is not `NULL`, it must point to an array of #az_log_classification,
```

Comment:
This is what CI is complaining about:
```suggestion
 * @details If \p classifications is not `NULL`, it must point to an array of #az_log_classification,
```